java/code/src/com/redhat/rhn/manager/audit/AuditManager.java

```diff
@@ -36,15 +36,18 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 /**
  * AuditManager
  */
-public class AuditManager /* extends BaseManager */ {
+public class AuditManager {
 
-    private static Logger log = LogManager.getLogger(AuditManager.class);
+    private static final Logger LOGGER = LogManager.getLogger(AuditManager.class);
+    private static Map<String, String[]> auditTypeMappings = null;
+    private static final Pattern AUDIT_FILENAME_PATTERN = Pattern.compile("audit-(\\d+)-(\\d+).parsed");
```

Comment:
Same as before, unless you are planning to use it on multiple places, define it close to where it is being used i-e in the method.